android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIAbstractTabGroup.java

```diff
@@ -115,6 +115,13 @@ public abstract class TiUIAbstractTabGroup extends TiUIView
 	 */
 	public abstract void updateTabBackgroundDrawable(int index);
 
+	/**
+	 * Material 3 active indicator color
```

Comment:
Does this map to a different color that we already have, e.g. `tintColor`?

Reply:
Yes, it's the M3 "Active Indicator" https://m3.material.io/components/navigation-bar/specs#2e9061a4-6dfb-41f0-b2d7-07e3b06b484f
![Screenshot_20241003-115212](https://github.com/user-attachments/assets/88362f81-cfc3-49db-8c3e-03e2df98e67e)

in this example activeTintColor is yellow and tintColor is blue

Reply:
current workaround:
```xml
  <style name="Widget.Material3.BottomNavigationView.ActiveIndicator" parent="">
        <item name="android:width">@dimen/m3_bottom_nav_item_active_indicator_width</item>
        <item name="android:height">@dimen/m3_bottom_nav_item_active_indicator_height</item>
        <item name="marginHorizontal">@dimen/m3_bottom_nav_item_active_indicator_margin_horizontal</item>
        <item name="shapeAppearance">@style/ShapeAppearance.Material3.NavigationBarView.ActiveIndicator</item>
        <item name="android:color">#ffff00</item>
    </style>
```

Reply:
I would expect that it defaults to a color that inherits from the base colors?

Reply:
the default color:

https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomnavigation/res/values/styles.xml#L105

and then 

https://github.com/material-components/material-components-android/blob/8f5a04c62b23aee01fa68aae8b467d7eb7c5e6ed/lib/java/com/google/android/material/bottomnavigation/res/values/tokens.xml#L49

deprecated default color `colorSecondaryContainer`?

People don't like to use the XMLs and prefer using Ti properties :smile:  That's why I've added that property (user request, I'm getting XML theme questions on slack all the time)

Reply:
Makes total sense then!